lms/djangoapps/teams/static/teams/js/teams_tab_factory.js

```diff
@@ -1,13 +1,13 @@
 ;(function (define) {
     'use strict';
 
-    define(['jquery', 'teams/js/views/teams_tab'],
-        function ($, TeamsTabView) {
-            return function () {
-                var view = new TeamsTabView({
-                    el: $('.team-tab-content')
-                });
-                view.render();
-            };
-        });
+    define(['jquery','teams/js/views/teams_tab'],
+           function ($, TeamsTabView) {
```

Comment:
Ah yes, did emacs do this? ;)

I'm not sure if there's a right or a wrong way to be indenting our JS when continuing function invocations. @andy-armstrong do you care much about this?